src/lib/OpenApi.ts

```
@@ -210,6 +210,73 @@
+    if (typeof tags === 'string') {
+      tags = [tags]
+    }
+
+    const paths = getPaths() ?? {}
+
+    const output = {}
+
+    for (const [path, methods] of Object.entries(paths)) {
+      for (const verb of httpVerbs) {
+        const operation = methods[verb]
+        if (operation && operation.tags && operation.tags.some(tag => tags.includes(tag))) {
```

Biome

[error] Change to an optional chain.

Unsafe fix: Change to an optional chain.


(lint/complexity/useOptionalChain)